xdsl/dialects/builtin.py

```diff
@@ -579,8 +579,8 @@ def from_float_list(
             type: VectorOrTensorOf[AnyFloat],
             data: List[float | AnyFloatAttr]) -> DenseIntOrFPElementsAttr:
```

Comment:
```suggestion
            data: List[int | float | AnyFloatAttr]) -> DenseIntOrFPElementsAttr:
```
since we now also accept ints

Reply:
Shouldn't we then also accept `AnyIntegerAttr` to be consistent?

Reply:
In that case, does it makes sense to change the name of the function as well? (Its current name is `from_float_list`)

Also, I might be a bit slow to respond in coming weeks since I am having my end-of-semester examinations at my university.

Reply:
We can rename it `create_dense_float` instead

Reply:
We should also change the `from_list` type to also allow `int` and `AnyIntegerAttr`

Reply:
> We can rename it create_dense_float instead,

Shall we also rename `from_int_list` to `create_dense_int` for uniformity then?

Reply:
Yes!